### PR TITLE
add brief for user workflow: adding users to projects/scenarios [MARXAN-1124]

### PR DESCRIPTION
**Pull request by hotzevzl**
https://vizzuality.atlassian.net/browse/MARXAN-1124

### CODE REVIEW

**Comment by hotzevzl** [comment]
@dhakelila would you mind having a quick look at the small bit of product documentation added in this PR? I had written up your summary (IIRC) and added some technical hints to document assumptions behind the feature as designed, so it should be already familiar, but I'd appreciate a cross-check if you can. thanks!

File: docs/features/role-based-access-control/managing-roles/brief.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+reject), to restrict the list of users that can be discovered to those within
+the parent organization/team of a project, etc.
+
+These possible features would improve the ergonomics of the workflow for adding
+users to projects or scenarios, but they also imply substantive additional
+complexity which should be avoided for a first, functional implementation.
+
+## Adding users to a project or a scenario - workflow
+
+- Project/scenario owners can click on a `+` (or similar) button next to the
```
Comment (dhakelila) [inline docs/features/role-based-access-control/managing-roles/brief.md:19]: I understood that we'll always be adding users to a project, not to a scenario.

File: docs/features/role-based-access-control/managing-roles/brief.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+desirable, in a future revision of the feature.
+
+- Once the desired user is found, a visual confirmation is given (that is, the
+  project/scenario owner needs to see that the email address they have typed is
+  that of an active user)
+
+- Project/scenario owner selects the user account just found, then:
+
+  - they are presented with a dropdown to select the desired role of the new
+    user within the relevant project/scenario
```
Comment (dhakelila) [inline docs/features/role-based-access-control/managing-roles/brief.md:47]: I'd add here the different options we are thinking of as roles: 
-Project contributor
-Project viewer (read-only)
-Solution-viewer  [nice-to-have]

**Review by dhakelila — COMMENTED** [review]
Looks good. Left a couple of little comments.
I'm using the users doc as reference to remember the roles/functionalities we had on mind
https://docs.google.com/document/d/16x4wQYTKov0JkO1ue6iQzJ3-RXpdgQ38ygGJWW86Zo0/edit#heading=h.uldt4yvqzw2o

File: docs/features/role-based-access-control/managing-roles/brief.md (hunk 1 above)
Comment (hotzevzl) [reply docs/features/role-based-access-control/managing-roles/brief.md]: you are right - I have removed these references (but also added a note about scenario-level role granting as a possible future development, if desired)

File: docs/features/role-based-access-control/managing-roles/brief.md (hunk 2 above)
Comment (hotzevzl) [reply docs/features/role-based-access-control/managing-roles/brief.md]: thanks - added too

**Pull request merged**